src/commands/utils/draft.ts

```diff
@@ -0,0 +1,75 @@
+import { Errorable, combine, failed } from "./errorable";
+import { DeploymentSpecType } from "../../webview-contract/webviewDefinitions/draft/types";
+import path from "path";
+import { ShellOptions, exec } from "./shell";
+import { parseJson } from "./json";
+import { dirSync } from "tmp";
+
+type DryRunOutput = {
+    variables: { [variableName: string]: string };
```

Comment:
I'm not aware of any documentation for Draft's `--dry-run` output.

This is only used for the `filesToWrite` property, which doesn't provide the value I was hoping for. The intention was that it could be used to predict that Draft commands would fail due to files already existing, but in fact Draft employs its own file existence checking which does not use this `filesToWrite` property.

Possible alternative approaches:
- Draft could provide a `force` option, which would tell it to overwrite existing files without any user interaction.
- Draft could be changed so that its `filesToWrite` dry-run output was consistent with its existing file checking.

Both these require discussion with the Draft team, and I don't think we're going to reach any conclusions right now. FYI: @qpetraroia.